src/Maestro/tests/DependencyUpdateErrorProcessor.Tests/CreateGithubIssueTests.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Maestro.Data;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace DependencyUpdateErrorProcessor.Tests
+{
+    public class CreateGithubIssueTest :DependencyUpdateErrorProcessorTests
+    {
+        [Fact]
+        public async Task UseInMemoryDatabaseTest()
+        {
+            var options = new DbContextOptionsBuilder<BuildAssetRegistryContext>()
+                .UseInMemoryDatabase(databaseName: "testDb")
+                .Options;
+            var repositoryBranchUpdate = new RepositoryBranchUpdateHistoryEntry()
+            {
+                Repository = "https://github.com/maestro-auth-test/maestro-test2",
+                Branch = "38",
+                Method = "SynchronizePullRequestAsync",
+                Success = false,
+                Timestamp = new DateTime(2020, 04, 01)
+            };
+            using (var context = new BuildAssetRegistryContext(Context.HostingEnvironment, options))
```

Comment:
Why is it a new context and not getting the one that you built in the options?

Reply:
I don't think this is the right place for this test.

Once you create a test BuldAssetRegistry, this test should be part of the tests for that class.